scripts/sct_straighten_spinalcord.py

```diff
@@ -116,7 +116,7 @@ def get_parser():
         metavar=Metavar.float,
         type=float,
         help='Size of the output FOV in the RL/AP plane, in mm. The resolution of the destination '
-             'image is the same as that of the source image (-i).',
+             'image is the same as that of the source image (-i). Default: 35.',
```

Comment:
Would it be relevant to display the default value by default in order to reduce maintenance etc?
eg https://stackoverflow.com/a/12151325/13306686
If so, I will open a new issue

Reply:
Yes that would be awesome!